packages/ui/src/kit/VRadio/VRadio.vue

```diff
@@ -1,23 +1,29 @@
 <template>
-  <label :class="vRadioCssClass">
-    <input
-      v-bind="$attrs"
-      :checked="value === modelValue"
-      type="radio"
-      class="v-radio-input"
-      @input="onInputChange"
+  <div :class="vRadioCssClass">
+    <label-molecule
+      class="v-radio-wrapper"
+      :tooltip-label="tooltipLabel"
```

Comment:
and tooltips for radio looks very strange for me

Reply:
Well, it's better than long text in one tooltip, because here may be more that 2 choices. I guess, the fact that tooltip aligned on baseline, play one of primary roles in "strange" feeling, maybe with `super` alignment it would be better.